specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
For clarity, I'd drop these. Otherwise questions arise about the need for literally hundreds of other tables like st_vrain_school_district_contest_table, which spans county boundaries but doesn't seem appropriate for state or national.
Accommodating the highly complex nature of Colorado district boundaries is a key concern for election software here.